jason_kelce.txt

```diff
@@ -0,0 +1,20 @@
+created_by: 
+domain: athlete
+seed_examples:
+- answer: |193
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!